locations/storefinders/elfsight.py

```diff
@@ -0,0 +1,44 @@
+from typing import Iterable
+from urllib.parse import unquote
+
+import chompjs
+from scrapy.http import JsonRequest, Request
+
+from locations.json_blob_spider import JSONBlobSpider
+
+
+class ElfsightSpider(JSONBlobSpider):
+    """
+    An Elfsight spider will be one of:
+    - https://shy.elfsight.com/p/boot/?callback=a&shop=(shop)&w=(api_key)
+    - https://core.service.elfsight.com/p/boot/?w=(api_key)
+    - Or embedded in data-elfsight-google-maps-options
+    """
+
+    host = None
```

Comment:
I decided not to in this case, as they have different meanings.
I suspect this Storefinder will split into three subclasses at some point in the future, and this will be redundant